src/component-tester.js

```diff
@@ -48,15 +48,19 @@ export class ComponentTester {
   create(): Promise<void> {
     return bootstrap(aurelia => {
       return Promise.resolve(this.configure(aurelia)).then(() => {
-        aurelia.use.globalResources(this._resources);
+        if (this._resources) {
+          aurelia.use.globalResources(this._resources);
+        }
         return aurelia.start().then(a => {
           let host = document.createElement('div');
           host.innerHTML = this._html;
           document.body.appendChild(host);
           aurelia.enhance(this._bindingContext, host);
           this._rootView = aurelia.root;
           this.element = host.firstElementChild;
-          this.viewModel = this.element.au.controller.viewModel;
+          if (aurelia.root.controllers.length) {
+            this.viewModel = aurelia.root.controllers[0].viewModel;
```

Comment:
Only the first controller's viewModel is exposed. An improvement might be to expose all controller's viewModels. In order to get the viewModels now would be to get hold of the `aurelia` object in the configure method and grab them from there.